docs/create-modify-element.mdx

```diff
@@ -9,6 +9,12 @@ import Admonition from "../components/Admonition"
 import Accordion from "../components/Accordion"
 import MDN from "../components/MDNBadge"
 
+:::note DOM interfaces
+Every kind of **DOM** node is represented by an interface based on **Node** interface.   
+**Element, Document and DocumentFragment** interfaces inherit from Node.  
+More about DOM interfaces [here](https://developer.mozilla.org/en-US/docs/Web/API/Document_Object_Model)
```

Comment:
Don't put everything in newline, can be distracting to readers. 
I understand that 2nd line is separate to draw reader's attention. Then, let's put 2nd and 3rd line together?

Reply:
Yup that can be done.